curvefs/src/client/s3/kvclient/kvclient_pool.h

```diff
@@ -0,0 +1,143 @@
+/*
+ *  Copyright (c) 2022 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: curve
+ * Created Date: 2022-09-22
+ * Author: YangFan (fansehep)
+ */
+
+#ifndef CURVEFS_SRC_CLIENT_S3_KVCLIENT_KVCLIENT_POOL_H_
+#define CURVEFS_SRC_CLIENT_S3_KVCLIENT_KVCLIENT_POOL_H_
+
+#include <assert.h>
+#include <glog/logging.h>
+#include <libmemcachedutil-1.0/pool.h>
+
+#include "curvefs/src/client/s3/kvclient/kvclient.h"
+
+namespace curvefs {
+
+namespace client {
+
+template <typename kvclient, typename pool>
+class KvClientPool {
```

Comment:
memcached_ pool_ St is a way of use. It is unnecessary to use a separate class to encapsulate it. It is recommended to put it in memcacheImp

Reply:
> memcached_ pool_ St is a way of use. It is unnecessary to use a separate class to encapsulate it. It is recommended to put it in memcacheImp

fix.